python_basics_lists_dictionaries/python_basics_lists_dictionaries.md

```diff
@@ -0,0 +1,453 @@
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+There are a couple of other collections, called [tuples](https://www.geeksforgeeks.org/python-tuples/) and [sets](https://www.geeksforgeeks.org/sets-in-python/); you can read more about those on your own if you're interested. 
+
+</div>
+
+
+## Lists
+
+**Lists** in Python are objects that contain a collection of multiple items. They are surrounded by square brackets `[ ]` and the items in the list are separated by commas:
+
+@sage
+<div class="python_link">
+<lia-keep>
+<script type="text/x-sage">
+
+produce = ["tomato", "squash", "apple", "cucumber", "peach", "spinach"]
+print(produce)
+
+</script>
+</lia-keep>
+</div>
+
+There are a few important characteristics of lists:
+
+* Lists can have duplicate values. An example might be the list `[3, 1, 4, 1, 5, 9]`, where the number 1 appears twice.
+
+* List items don't all have to be of the same type (you can have a mix of strings and numbers, for example, or even a list of lists). `[3.14, "pi", False, ["number", "string", "boolean"]]` is an example of a list that contains a float, a string, a boolean value, and a list as its elements.
+
+* Lists are **ordered**: List items remain in the position that they are entered into the list, and you can access any item by its position, or **index**.
+
+    <div class = "important">
+    <b style="color: rgb(var(--color-highlight));">Important note</b><br>
+
+    An **index** in Python refers to a value's position in a sequence. The sequence can be a collection object like a list or even a string (which you can think of as a sequence of characters). The important thing to remember is that in Python, **indices start at 0**. This means that the index of the first item in a sequence is `0`! This is not always the case in other programming languages.
+
+    To access an item by its index, you use **subsetting notation**, `sequence[index]`. To get the first letter of the string "Hello" (strings can be thought of as a sequence of characters), for example, you would use the code `"Hello"[0]`. Notice that this is another use of square brackets!
+
+    </div>
+
+* Lists are **changeable**; you can add, remove, and edit items in a list. You can use list [methods](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/python_basics_variables_functions_methods/python_basics_variables_functions_methods.md#6) to do this: 
+
+    * `.pop()` removes a specific element by **index**.
+
+     @sage
+     <div class="python_link">
+     <lia-keep>
+     <script type="text/x-sage">
+
+     print(produce.pop(0))
+
+     </script>
+     </lia-keep>
+     </div>
+
+   * `.remove()` removes a specific element by **value**.
+
+    @sage
+    <div class="python_link">
+    <lia-keep>
+    <script type="text/x-sage">
+
+    produce.remove("apple")
+    print(produce)
```

Comment:
```suggestion
print(produce)
```
Right now it is indenting this line inside the cell, which we don't want, I think this will keep the cell itself indented without indenting the line.